cookbooks/aws-parallelcluster-slurm/recipes/test/mock_munge_key.rb

```diff
@@ -1,4 +1,4 @@
-munge_user_dir = "/home/#{node['cluster']['cluster_user']}/.munge"
+munge_user_dir = "#{node['cluster']['shared_dir']}/.munge"
```

Comment:
I'm not too sure what this file is for (I guess for some kitchen tests that still need to be developed), but is it correct that this uses the compute node shared folder?

Reply:
I am not sure too. Previously, both Login Nodes and Compute Nodes fetch munge key from `"/home/#{node['cluster']['cluster_user']}/.munge"`. 
But maybe now we should also mock another 
`munge_user_dir_login = "#{node['cluster']['shared_dir_login']}/.munge"`
and
```
file "#{munge_user_dir_login}/.munge.key" do
  content 'munge-key'
  owner node['cluster']['munge']['user']
  group node['cluster']['munge']['group']
end
```